libraries/botbuilder-dialogs-adaptive/src/index.ts

```diff
@@ -29,6 +29,7 @@ export * from './resourceExtensions';
 export * from './schemaHelper';
 export * from './selectors';
 export * from './skillExtensions';
+export * from './telemetryExtensions';
```

Comment:
Please see previous comment about the class name and update the filename accordingly.